tests/data_access/models/user_model_test.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+
+import src.data_access.models.user_model as user_model
+from src.data_access.models.user_model import User
+
+
+@pytest.fixture
+def mock_generate_password_hash(mocker):
+    return mocker.patch.object(
+        user_model, "generate_password_hash"
```

Comment:
u can remove the import of user_model and use User - mocker.patch.object(User, "generate_password_hash")

Reply:
User is the class and user_model is the module, so the function im patching belongs to user_model module